components/Radio/Interface.ts

```diff
@@ -0,0 +1,70 @@
+import { CSSProperties, ReactNode } from "react"
+
+export interface RadioWrapperProps {
+  prefixCls?: string
+  className?: string
+  style?: CSSProperties
+  defaultChecked?: boolean
+  checked?: boolean
+  disabled?: boolean
+  onFocus?: (e) => void | any
+  onBlur?: (e) => void | any
+  onChange?: (e) => void | any
+  onClick?: (e) => void | any
+  tabIndex?: number
+  readOnly?: boolean
+  autoFocus?: boolean
+  value?: any
+  groupValue?: string | number
+  setValue?: (e) => void | any
+  size?: string
+}
+
+type size = "small" | "medium"
+
+export interface RadioChangeEvent {
+  target: RadioChangeEventTarget
+  stopPropagation: () => void
+  preventDefault: () => void
+  nativeEvent: MouseEvent
+}
+
+export interface RadioGroupProps {
+  /** Value of the radio group */
+  value?: any
+  /** Custom styles for each radio button */
+  style?: CSSProperties
+  /** Function to call when radio is changed */
+  onChange?: (e: RadioChangeEvent) => void
+  /** default value of the group */
+  defaultValue?: any
```

Comment:
Group should not have vaue or defaultValue props.

Reply:
Removed value, refer above for default value.